_posts/intro/2014-02-01-raspberry-pi-quick-start.md

```diff
@@ -50,49 +50,38 @@ following steps:
 
     <pre>sudo apt-get purge dhcpcd5</pre>
 
-3. NetworkManager conflicts with Kura network management, make sure that it is
-    not installed performing the following command:
+3. After removing dhcpcd5 package you need to enable network interface(s) and ensure internet access. Follow the instructions <a href="https://raspberrypi.stackexchange.com/questions/37920/how-do-i-set-up-networking-wifi-static-ip-address/74428#74428" target="_blank">here</a>.
```

Comment:
So, after removing the dhcpcd5, the RPi is configured with static addresses. The network manager than comes with Raspiabian is not explicitly disabled, since Kura remove it during installation. Is this correct?

Reply:
im not sure. 
with the latest rasbian network-manager package is not installed by default so if you try to remove it "sudo apt-get remove network-manager", you will get "Package 'network-manager' is not installed, so not removed"

Reply:
Ah ok. What is the default service that manage the networking with the new Rasbian? I mean, Kura acts as a network manager, so if the OS has one active, the two conflict and the latter has to be disabled before installing Kura.

Reply:
I think dhcpcd is default network manager. if we disable networking before instaling kura, rpi will boot without any interface enabled.  im not sure that kura disables networking after installation but if I look at the statuses - they are not active (dhcpcd.service and networking.service)

Reply:
@pierantoniomerlino Are you ok to merge then? Or do you want @mstankovic to perform more tests?

Reply:
For me it's ok. I'll merge it.